include/camp/list/list.hpp

```diff
@@ -8,6 +8,7 @@ namespace camp
 template <typename... Ts>
 struct list {
   using type = list;
+  static constexpr size_t size = sizeof...(Ts);
```

Comment:
Why is?

I don't hate this necessarily, but `camp::size<L>::value` would do this, and only creates the memory when its used rather than for every list type.

Reply:
fair enough.  I will remove that and go with the camp::size<> instead.

Reply:
Ok.  Also, as part of the at_v thing I'm going to add an f-suffixed version of some of these that wrap them in a function to avoid the ::value.  I really wish we could just use c++14 template variables for this actually, maybe it's worth testing all of our target compilers to see if they handle that ok, but either way that should be a bit less noisy soon.